presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestEliminateCrossJoins.java

```diff
@@ -65,16 +64,14 @@ public void testEliminateCrossJoin()
                 .setSystemProperty(REORDER_JOINS, "true")
                 .on(crossJoinAndJoin(INNER))
                 .matches(
-                        project(
```

Comment:
The previous code was introducing a project only because the output order had changed, which was not meaningful.  The new plan is better.